.buildkite/cpu_scaling_pipeline/pipeline.yml

```diff
@@ -0,0 +1,550 @@
+agents:
+  queue: new-central
+  slurm_mem: 8G
+  modules: climacommon/2024_12_16
+
+env:
+  JULIA_LOAD_PATH: "${JULIA_LOAD_PATH}:${BUILDKITE_BUILD_CHECKOUT_PATH}/.buildkite"
+  OPENBLAS_NUM_THREADS: 1
+  SLURM_KILL_BAD_EXIT: 1
+  JULIA_NVTX_CALLBACKS: gc
+  JULIA_MAX_NUM_PRECOMPILE_FILES: 100
+  JULIA_DEPOT_PATH: "${BUILDKITE_BUILD_PATH}/${BUILDKITE_PIPELINE_SLUG}/depot/default"
+  CONFIG_PATH: "config/model_configs"
+  MBW_SCALING_CONFIG_PATH: "config/mbw_scaling_configs"
+  CPU_TYPE: "icelake"
+
+steps:
+  - label: "init :computer:"
+    key: "init_cpu_env"
+    concurrency: 1
+    concurrency_group: 'depot/climaatmos-ci'
```

Comment:
```suggestion
```
These are not needed and will slow the CPU scaling pipeline down if climaatmos-ci is instantiating.